MediaBrowser.WebDashboard/dashboard-ui/addplugin.html

```diff
@@ -45,14 +45,14 @@ <h2 class="sectionTitle">${HeaderInstall}</h2>
 
                                     <p>${MessagePluginRequiresSubscription}</p>
 
-                                    <p><a is="emby-linkbutton" class="button-link" href="https://emby.media/premiere" target="_blank">${LinkLearnMoreAboutSubscription}</a></p>
+                                    <p><a is="emby-linkbutton" class="button-link" href="https://github.com/jellyfin/jellyfin" target="_blank">${LinkLearnMoreAboutSubscription}</a></p>
```

Comment:
Shouldn't this be removed completely? There is not going to be any subscription to Jellyfin, right?